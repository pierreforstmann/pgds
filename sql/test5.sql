--
-- test5.sql
--
\pset null (null)
--
create table t500(x int, y date) partition by range(y);

create table t500_2020 partition of t500
for values from ('2020-01-01') to ('2020-12-31');
--
--
select min(y) from t500;
--
create table t500_2021 partition of t500
for values from ('2021-01-01') to ('2021-12-31');
--
create table t500_2022 partition of t500
for values from ('2022-01-01') to ('2022-12-31');
--
--
select max(y) from t500;
