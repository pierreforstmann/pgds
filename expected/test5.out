--
-- test5.sql
--
\pset null (null)
--
create table t500(x int, y date) partition by range(y);
create table t500_2020 partition of t500
for values from ('2020-01-01') to ('2020-12-31');
--
--
select min(y) from t500;
INFO:  analyzing "public.t500" inheritance tree
INFO:  analyzing "public.t500_2020"
INFO:  "t500_2020": scanned 0 of 0 pages, containing 0 live rows and 0 dead rows; 0 rows in sample, 0 estimated total rows
  min   
--------
 (null)
(1 row)

--
create table t500_2021 partition of t500
for values from ('2021-01-01') to ('2021-12-31');
--
create table t500_2022 partition of t500
for values from ('2022-01-01') to ('2022-12-31');
--
--
select max(y) from t500;
INFO:  analyzing "public.t500" inheritance tree
INFO:  analyzing "public.t500_2020"
INFO:  "t500_2020": scanned 0 of 0 pages, containing 0 live rows and 0 dead rows; 0 rows in sample, 0 estimated total rows
INFO:  analyzing "public.t500_2021"
INFO:  "t500_2021": scanned 0 of 0 pages, containing 0 live rows and 0 dead rows; 0 rows in sample, 0 estimated total rows
INFO:  analyzing "public.t500_2022"
INFO:  "t500_2022": scanned 0 of 0 pages, containing 0 live rows and 0 dead rows; 0 rows in sample, 0 estimated total rows
  max   
--------
 (null)
(1 row)

